prime-router/src/main/kotlin/fhirengine/translation/hl7/schema/ConfigSchemaReader.kt

```diff
@@ -38,16 +63,31 @@ object ConfigSchemaReader : Logging {
             }
             val depSchemaFolder = "$folder/${FilenameUtils.getPath(schemaList.last().extends)}"
             val depSchemaName = FilenameUtils.getName(schemaList.last().extends)
-            schemaList.add(readSchemaTreeFromFile(depSchemaName, depSchemaFolder, schemaClass = schemaClass))
+            schemaList.add(readSchemaTreeRelative(depSchemaName, depSchemaFolder, schemaClass = schemaClass))
         }
+        return schemaList
+    }
 
-        // Now merge the parent with all the child schemas
-        val mergedSchema = mergeSchemas(schemaList)
-
-        if (!mergedSchema.isValid()) {
-            throw SchemaException("Invalid schema $schemaName: \n${mergedSchema.errors.joinToString("\n")}")
+    /**
+     * Read a schema from a [URI] using the scheme to determine who to read the schema
```

Comment:
This sentence doesn't make sense? `who` -> `how`?

Reply:
Will update